src/lesson02/Task04.java

```diff
@@ -0,0 +1,19 @@
+package lesson02;
+import java.util.Scanner;
+public class Task04 {
+    //programma perovodu Fahrenheit to Celsius;
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        double degreeFahrenheit;
+        System.out.print("Vvedit znachennya po Fahrenheit: " + "degreeFahrenheit=");
```

Comment:
тут тоді розділяти не потрібно 
```suggestion
        double degreeFahrenheit;
        System.out.print("Vvedit znachennya po Fahrenheit: degreeFahrenheit=");
```